Makefile

```diff
@@ -0,0 +1,30 @@
+SHELL := /bin/bash
+
+.PHONY: .rocks
+.rocks: graphql-scm-1.rockspec Makefile
+		tarantoolctl rocks make
+		tarantoolctl rocks install luatest 0.5.5
+		tarantoolctl rocks install luacov 0.13.0
+		tarantoolctl rocks install luacheck 0.26.0
+
+.PHONY: lint
+lint:
+		if [ ! -d ".rocks" ]; then make .rocks; fi
+		.rocks/bin/luacheck .
+
+.PHONY: test
+test:
+		if [ ! -d ".rocks" ]; then make .rocks; fi
+		rm -f tmp/luacov*
+		.rocks/bin/luatest --verbose --coverage --shuffle group
+		.rocks/bin/luacov . && grep -A999 '^Summary' tmp/luacov.report.out
+
+.PHONY: clean
+clean:
+		rm -rf .rocks
+
+.PHONY: build
+build:	
+		if [ ! -d ".rocks" ]; then make .rocks; fi
+		tarantoolctl rocks make
+		tarantoolctl rocks pack graphql scm-1	
```

Comment:
Why we may need to create .all.rock for scm-1?

Reply:
I borrowed it from original PR, but I also were in several situations when I have found a bug in a rock package with my application and after drafting a fix I wanted to ensure that issue with my application is resolved by installing scm-1 and running some tests (for me it was tarantool/metrics).

Reply:
Okay, but why do you need to pack it for that? Do you use a rock tarball for deployment within an application?

Reply:
Well, to install `myrock` to `app/.rocks`, I use `tarantoolctl rocks install ./myrock-scm-1.all.rock`